geomesa-spark-sql/geomesa-spark-sql-core/src/main/scala/org/apache/spark/sql/SQLRules.scala

```diff
@@ -0,0 +1,152 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.apache.spark.sql
```

Comment:
Spark w/ 2.0 release reduced visibility of some API.  This is acknowledged by authors as non-ideal but is required at this time...